pkg/controllers/user/cis/rbac.go

```diff
@@ -0,0 +1,134 @@
+package cis
+
+import (
+	"fmt"
+
+	"github.com/rancher/rancher/pkg/namespace"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	rbacv1 "github.com/rancher/types/apis/rbac.authorization.k8s.io/v1"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	cisReadOnlyRoleName = "cis-readonly"
+)
+
+type cisScanRBACHandler struct {
+	roles             rbacv1.RoleInterface
+	roleLister        rbacv1.RoleLister
+	roleBindings      rbacv1.RoleBindingInterface
+	roleBindingLister rbacv1.RoleBindingLister
+}
+
+func (rh *cisScanRBACHandler) Sync(key string, crtb *v3.ClusterRoleTemplateBinding) (runtime.Object, error) {
+	if crtb == nil {
+		return nil, nil
+	}
+	if crtb.DeletionTimestamp != nil {
+		return rh.handleDelete(key, crtb)
+	}
+	if crtb.RoleTemplateName != "cluster-owner" || crtb.ClusterName == "local" {
```

Comment:
This will eventually bite us, but it might be good enough for now. You can't hard code the check to a role name. Operators can create their own RoleTemplates that are functionally equivalent to cluster-owner (or close enough).

Reply:
The correct solution would be to do something similar to
https://github.com/rancher/rancher/blob/4a72409ef5122d3902faee2b3e051799573ad179/pkg/controllers/management/auth/manager.go#L379-L382
which calculates whether you should get a certain permission based on the rules in the RoleTemplate, rather than the name of the role.

In fact, if you were to parameterize the namespace used in that method, you _might_  be able to re-use it.

Reply:
@deniseschannon I want to leave the "call" for this one up to you as it is as much about time and effort as anything else. tl;dr is we are hard coding a check to a roleTemplate name `cluster-owner`, but users can define their own. If they have, then they wont get the permission that lets them see cisconfigs and cisbenchmarkversions.

Of the roles that people actually replace, I would expect cluster-owner to be fairly low on the list because it simply means you have full operational control over the cluster. I do recall someone wanting to create, basically, a "security read-only" role. That could be used to allow a sec team to view a cluster, but not modify it. With that use case, they wouldn't be able to see the two mentioned resources with how this is coded now.

Reply:
Basically only cluster owners can see these resources and there is no ability to add them to a custom role? @cjellick @leodotcloud

If that is the case, then this is fine for for 2.4.0 and we could add in the ability to manage cisconfigs/cisbenchmarkversions as a separate role in a later release.

Reply:
@deniseschannon that is correct. Only cluster owners can see these CIS info related resources.